pkg/api/customization/cluster/formatter.go

```diff
@@ -47,6 +47,14 @@ func (f *Formatter) Formatter(request *types.APIContext, resource *types.RawReso
 			resource.AddAction(request, v3.ClusterActionBackupEtcd)
 			resource.AddAction(request, v3.ClusterActionRestoreFromEtcdBackup)
 		}
+		isActiveCluster := false
+		if resource.Values["state"] == "active" {
+			isActiveCluster = true
+		}
+		isWindowsCluster := false
+		if resource.Values["windowsPreferedCluster"] == true {
+			isWindowsCluster = true
+		}
```

Comment:
Should we also add the Windows check to the action handler to deny calling this action on the cluster at all

Reply:
good point, will add.